.github/dependabot.yml

```diff
@@ -0,0 +1,19 @@
+version: 2
+updates:
+  - package-ecosystem: npm
```

Comment:
By the way, we can also maintain dependencies for GitHub Actions here

```yml
- package-ecosystem: "github-actions"
  directory: "/"
  schedule:
    interval: "weekly"
```

And there is a way to auto-approve and auto-merge Dependabot PRs. E.g., it would look like this for semver `patch` releases

```yml
name: Dependabot auto-merge
on: pull_request

permissions: write-all

jobs:
  dependabot:
    runs-on: ubuntu-latest
    if: ${{ github.actor == 'dependabot[bot]' }}
    steps:
      - name: Dependabot metadata
        id: metadata
        uses: dependabot/fetch-metadata@v1.3.3
        with:
          github-token: "${{ secrets.GITHUB_TOKEN }}"
      - name: Approve Dependabot PRs
        if: ${{steps.metadata.outputs.update-type == 'version-update:semver-patch'}}
        run: gh pr review --approve "$PR_URL"
        env:
          PR_URL: ${{github.event.pull_request.html_url}}
          GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
      - name: Enable auto-merge for Dependabot PRs
        if: ${{steps.metadata.outputs.update-type == 'version-update:semver-patch'}}
        run: gh pr merge --auto --squash "$PR_URL"
        env:
          PR_URL: ${{github.event.pull_request.html_url}}
          GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
```

Reply:
For GitHub Actions, we specify dependencies through NPM too here https://github.com/mapbox/mapbox-gl-js/blob/main/.github/actions/package.json — and it should get picked up by the current Dependabot config too. For auto-merging, let's hold up on that and revisit later if it gets annoying.